leapp/libraries/stdlib/__init__.py

```diff
@@ -0,0 +1,25 @@
+"""
+:py:mod:`leapp.libraries.stdlib` represents a location for functions that otherwise would be defined multiple times across leapp actors
+ and at the same time are really useful for other actors.
+
+Example:
+    If the function call() is called from an actor
```

Comment:
This example does not really contribute to anything. This is basic python functionality, I would not bother making an example.